docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -533,7 +533,7 @@ p2p-port="1789"
 ```
 
 Specifies the port on which P2P peer discovery listens.
```

Comment:
This port is used for both discovery and wire protocol communication.  Maybe change this to something like: "Specifies the UDP port on which the devP2P discovery protocol listens and the TCP port on which the devP2P wire protocol listens."  Or something simpler like "Specifies p2p listening ports (UDP and TCP)".

Reply:
Updated